src/content/en/updates/2017/06/object-rest-spread.md

```diff
@@ -0,0 +1,84 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Object rest & spread properties
+
+{# wf_updated_on: 2017-06-05 #}
+{# wf_published_on: 2017-06-05 #}
+{# wf_tags: javascript #}
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+# Object rest & spread properties {: .page-title }
+
+{% include "web/_shared/contributors/mathiasbynens.html" %}
+
+Good ol’ ECMAScript 2015 introduced _rest elements_ for array destructuring
+assignment and _spread elements_ for array literals.
+
+    // Rest elements for array destructuring assignment:
+    const primes = [2, 3, 5, 7, 11];
+    const [first, second, ...rest] = primes;
+    console.log(first); // 2
+    console.log(second); // 3
+    console.log(rest); // [5, 7, 11]
+
+    // Spread elements for array literals:
+    const primesCopy = [first, second, ...rest];
+    console.log(primesCopy); // [2, 3, 5, 7, 11]
+
+These ES2015 features have been supported since Chrome 46 and Chrome 47,
+respectively.
+
+[A stage 3 proposal](https://github.com/tc39/proposal-object-rest-spread)
```

Comment:
This is a brand new feature that wasn’t part of ES2015. It’s not even part of the draft spec right now!

The proposal is at stage 3 of [the TC39 standardization process](https://tc39.github.io/process-document/). The proposal link explains it, so I don’t think we should repeat this here, especially as we’ll be writing about more such proposals in the future. WDYT?